learning_paths/models.py

```diff
@@ -20,6 +22,7 @@ class LearningPath(TimeStampedModel):
     .. no_pii:
     """
 
+    uuid = models.UUIDField(blank=True, default=uuid4, editable=False, unique=True)
```

Comment:
Definitely!

Reply:
@CefBoud Is blank=True only to facilitate the migration or is there another reason for it?

Reply:
@ArturGaspar 
> Is blank=True only to facilitate the migration or is there another reason for it

Not really. Since this field was added to mirror the course-discovery's Program[ UUID field](https://github.com/openedx/course-discovery/blob/e873415fbeee95535d354bf7547756d7a8af76ee/course_discovery/apps/course_metadata/models.py#L3085), I simply copied it as-is. 
Now that you mention it, I am not really sure why it has `blank = True`. But I thought it simpler to replicate the conf.